common/thorhelper/thorcommon.ipp

```diff
@@ -381,6 +384,10 @@ public:
     {
         return new CPrefixedRowDeserializer(offset, original->createInternalDeserializer(ctx, activityId));
     }
+    virtual void process(const byte * self, IFieldProcessor & target, unsigned from, unsigned to)
+    {
```

Comment:
@ghalliday This definition for process()  seems to be different from the previous base class definition for process().  If this ticket simply removes default implementation of virtual functions, then I would have thought that this definition should reflect base class definition?

Reply:
You're right, this is a bug fix added at the same time.  I will move them to a different PR.